pkg/sql/rename_table.go

```diff
@@ -229,3 +231,16 @@ func (p *planner) dependentViewError(
 			op, typeName, objName, viewName),
 		"you can drop %s instead.", viewName)
 }
+
+// writeNameKeyToBatch writes a name key to a batch and runs the batch.
+func (p *planner) writeNameKeyToBatch(
+	ctx context.Context, key roachpb.Key, ID descpb.ID, tracingEnabled bool,
+) error {
+	b := &kv.Batch{}
+	if tracingEnabled {
```

Comment:
you have the planner, just check there whether tracing is enabled, rather than taking it as a parameter

Reply:
Done